### ISSUE TITLE
not honoring -z option flag for duration

### ISSUE THREAD

**Issue by justinlarose** [bojand/grpcannon#14]
OS: Mac OS
grpcannon version: 0.4.1

When running grpcannon using the  `-z` option flag, grpcannon is still defaulting to running 200 requests instead of honoring the time duration value.

example:
```
grpcannon -z 1m -cert cert.pem --proto employeedirectory.proto -call directory.position.GetEmployee -d '{"name":"Steve", "position":"doctor"}' -M metadata.json jobs.search.com:8080
```

**Comment by bojand** [comment]
Are the 200 requests done before 60 seconds (`1m`) are up? I have a hard time believing that 200 requests at the default concurrency of 50 would take longer than 1 minute but I do not know the specifics of your service. Also the data in `-d` parameter doesn't seem to be properly formatted JSON?

I just downloaded the 0.4.1 and tried it and it seems to be working fine:

```
~/Downloads/grpcannon_0.4.1_darwin_amd64 ./grpcannon -z 1m -proto $GOPATH/src/github.com/bojand/grpcannon/testdata/greeter.proto -call helloworld.Greeter.SayHello -d '{"name":"Bob"}' -n 1000000 -c 1 0.0.0.0:50051
Stopped due to test timeout after 1m0s

Summary:
  Count:	365866
  Total:	60002.42 ms
  Slowest:	10.02 ms
  Fastest:	0.10 ms
  Average:	0.15 ms
  Requests/sec:	6097.52

Response time histogram:
  0.098 [1]	|
  1.091 [365741]	|∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎
  2.083 [87]	|
  3.076 [20]	|
  4.069 [12]	|
  5.061 [2]	|
  6.054 [2]	|
  7.046 [0]	|
  8.039 [0]	|
  9.032 [0]	|
  10.024 [1]	|

Latency distribution:
  10% in 0.13 ms
  25% in 0.13 ms
  50% in 0.14 ms
  75% in 0.15 ms
  90% in 0.19 ms
  95% in 0.21 ms
  99% in 0.32 ms
Status code distribution:
  [OK]	365866 responses


~/Downloads/grpcannon_0.4.1_darwin_amd64 ./grpcannon -v
0.4.1
~/Downloads/grpcannon_0.4.1_darwin_amd64
```

You can see

> Stopped due to test timeout after 1m0s

and the fact that the total time is `60002 ms` (1 minute) and the count is only `365866`.

**Comment by justinlarose** [comment]
My JSON is not properly formatted. I'll update it. 

Thank you for the clarification. I think I was confused by the flag description. It states `If duration is specified, n is ignored.`. I understood this to mean that the test would run for the duration specified and a value for `-n` did not need to be supplied. But it appears that `-n` is not ignored and should be set to a value where the test completion duration should exceed duration set by `-z`.

**Comment by bojand** [comment]
Oh I see... Hmm yes that may be a bug either in the description of `z` or handing of `z` and `n` like you state. Thanks for pointing that out.

**Comment by bojand** [comment]
Hello, 

I have changed the `-z` option to behave as the help documentation specified. Additionally now we have `-x` option which is the behaviour of what `-z` used to be as demonstrated above. Released as [0.5.0](https://github.com/bojand/grpcannon/releases/tag/v0.5.0).

**Comment by justinlarose** [comment]
thanks for fixing this so quickly!
